change/@fluentui-react-card-c7cf43df-6a47-4745-9d92-9064166abdb0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31681&commitId=0a6dc0180eceef13d4609120e84ca19dc3f697a7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Added|
|Avatar Converged.basic.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.CardHeader.normal.chromium.png|63608| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>